scalacheck/src/test/scala/magnolify/scalacheck/test/ArbitraryDerivationSuite.scala

```diff
@@ -59,6 +59,7 @@ class ArbitraryDerivationSuite extends MagnolifySuite {
     import Collections._
     test[Repeated]
     test((c: Collections) => (c.a.toList, c.l, c.v))
+    // test[MoreCollections]
```

Comment:
can this be uncommented?

Reply:
There is an implicit problem with `scalacheck` and 'exotic' collections with scala 2.12. It does not create the required `Buildable` to generate the corresponding `Arbitrary`.